typescript/src/lib/services/service.ts

```diff
@@ -7,3 +7,21 @@
 The identifier used for services with the network.
 */
 export type ServiceID = string;
+
+export class SharedDirectory {
```

Comment:
Same comment as Go - IMO we should move this to its own class

Reply:
EDIT: Oh wait, this actually already is in its own class down below - this one needs to be removed